docs/tutorials/gstreamer.mdx

```diff
@@ -1 +1,100 @@
+---
+sidebar_position: 4
+---
 
+# Build GStreamer pipeline with opencv-python 
+
+This tutorial demonstrates how to use OpenCV with GStreamer to capture video from cameras. It supports different camera configurations, pixel formats, and frame size.
+
```

Comment:
Can you shortly introduce appsink and opencv-python with gstreamer here? Like "while you can build and run pipeline with gstreamer API, this element and opencv api combination allows you to get an output of gstreamer pipeline" or something like that.

Reply:
done

Reply:
confirmed